composer.json

```diff
@@ -7,8 +7,11 @@
             "Codelicia\\XulietaJson\\": "src/"
         }
     },
-    "require-dev": {
-        "doctrine/coding-standard": "^8.2.0",
+    "require": {
+        "php": ">=7.4 || ~8.0",
         "codelicia/xulieta": "^0.1.3"
```

Comment:
`codelicia/xulieta` should stay under `require-dev`.
For plugins, we just need it for development.

Reply:
So requiring `codelicia/xulieta-json` will not be enough? Users should always add the main package?

Reply:
That is our idea. So people can plug in only what they need.

Reply:
> what they need.

Is there such a use case when the main package is not needed?
I maintain several plugins and all of them depend on the main package.

Reply:
Ah! I was talking about the plugins 😊

I can't think of a case where the main package is not required, but I want to minimize as much as possible the risk of conflict, so we can try different versions.

Reply:
Reverted with sadness.

Reply:
DCO: I'm sorry, I do it only here on GitHub's web interface...